src/scale-manager/README.md

```diff
@@ -1,23 +1,115 @@
 # Scale Manager
-A utility class that listens for resize events.
-
-There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
+A utility class that listens for resize events and calculates the width and height your game should be to fit within the screen without stretching or squishing the game. 
 
 ```javascript
-var scaleManager = new ScaleManager(function(resizeData) {
-  console.log('This is called on window resize');
+let scaleManager = new ScaleManager({
+  width: 1320,
+  height: 780,
+  callback: function(resizeData) {
+    console.log('This is called on window resize');
 
-  console.log('Window width is', resizeData.width);
-  console.log('Window height is', resizeData.height);
-  console.log('Window aspect ratio is', resizeData.ratio);
+    console.log('game width is', resizeData.width);
+    console.log('game height is', resizeData.height);
+    console.log('game scale is', resizeData.scale);
+  }
 });
 ```
 
-The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the
-existing listener.
+The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the existing listener. 
 
 ```javascript
 scaleManager.enable(function(resizeData) {
   console.log('The old listener is replaced with this one!', resizeData);
 });
 ```
+
+**Note:** There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
```

Comment:
I would update the text to better reflect the known bug: "to solve issues in iOS where the window dimensions provided by `window.innerWidth` and `window.innerHeight` do not match the actual dimensions of the device". I think would also be helpful to link to the WebKit bug report (althought I can't seem to find it for some reason now...)